guides/common/modules/ref_system-requirements-quickstart.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="system-requirements"]
+= System requirements
+
+//The requirements in this section must match the full requirements in the installation guide.
+The following list shows the most critical requirements common for most scenarios.
+For a full list, see {InstallingServerDocURL}[_{InstallingServerDocTitle}_].
+
+* The following operating systems are supported for deploying {Project}:
++
+.Operating systems supported for deploying {Project}
```

Comment:
On https://theforeman.org/manuals/3.15/quickstart_guide.html we have a simple bullet list instead of a table. Do we really need a full table for 2 columns?